tests/integration/llm/prepare.py

```diff
@@ -25,13 +25,32 @@
         "option.s3url": "s3://djl-llm/opt-6b7/",
         "option.tensor_parallel_degree": 4,
         "option.task": "text-generation",
-        "option.dtype": "float16"
+        "option.dtype": "float16",
+        "option.save_mp_checkpoint_path": "/opt/ml/input/data/training/partition-test"
     },
-    "gpt-neox-20b": {
```

Comment:
why swapping the model?

Reply:
Did not swap it. This was added in prepare.py but never used in llm_intergation.yaml.

Reply:
Do you want to add gptneox as well in the test cases?